README.md

```diff
@@ -286,27 +311,30 @@ entire scope of the Route; instead its scope is _explicitly defined_ at the
 location where the Component is invoked. This has the advantage of making
 Components very modular (and consequently, more interchangeable and re-usable).
 
-The change-over from Controllers and Views to Components is in process but is
-not complete. Both Controllers and Views are deprecated, but _Components are not
-yet 'routable'_ (though that change will probably be coming soon). The
-work-around is straight-forward:
+The change-over from Controllers and Views (Ember 1.0) to Components (Ember 2.0)
```

Comment:
"changeover" is actually not hyphenated: http://www.dictionary.com/browse/changeover